pom.xml

```diff
@@ -129,7 +129,7 @@ Wisconsin-Madison, Broad Institute of MIT and Harvard, and Max Planck
 Institute of Molecular Cell Biology and Genetics.</license.copyrightOwners>
 		<license.projectName>ImageJ software for multidimensional image processing and analysis.</license.projectName>
 
-		<scijava.jvm.version>1.6</scijava.jvm.version>
+		<scijava.jvm.version>1.8</scijava.jvm.version>
```

Comment:
Why? (If we do need this: we can just remove this line, rather than changing it, since 1.8 is the default these days.)
Edit: Oh, it was for the lambda below. I think we should change the code not to use a lambda, and instead be 1.6-compliant. Sorry about that, but it would keep the door open for a smarter launcher that complains when the version of Java is too old, rather than simply exploding.